mimic/rest/maas_api.py

```diff
@@ -0,0 +1,960 @@
+        self.alarms_list = []
+        self.notifications_list = [{'id': 'ntTechnicalContactsEmail',
+                                    'label': 'Email All Technical Contacts',
+                                    'created_at': time.time(),
+                                    'updated_at': time.time(),
+                                    'metadata': None,
+                                    'type': 'technicalContactsEmail',
+                                    'details': None}]
+        self.notificationplans_list = [{'id': 'npTechnicalContactsEmail',
+                                        'label': 'Technical Contacts - Email',
+                                        'critical_state': [], 'warning_state': [],
+                                        'ok_state': [], 'metadata': None}]
+        self.notificationtypes_list = [{'id': 'webhook', 'fields': [{'name': 'url',
+                                                                     'optional': False,
+                                                                     'description': 'An HTTP or \
+                                                                      HTTPS URL to POST to'}]},
+                                       {'id': 'email', 'fields': [{'name': 'address',
+                                                                   'optional': False,
+                                                                   'description': 'Email \
+                                                                    address to send notifications to'}]},
+                                       {'id': 'pagerduty', 'fields': [{'name': 'service_key',
+                                                                       'optional': False,
+                                                                       'description': 'The PagerDuty \
+                                                                        service key to use.'}]},
+                                       {'id': 'sms', 'fields': [{'name': 'phone_number',
+                                                                 'optional': False,
+                                                                 'description': 'Phone number to send \
+                                                                  the notification to, \
+                                                                  with leading + and country \
+                                                                  code (E.164 format)'}]}]
+        self.suppressions_list = []
+
+
+def createEntity(params):
+    """
+    Returns a dictionary representing an entity
+    """
+    params = collections.defaultdict(lambda: '', params)
+    newentity = {}
+    newentity['label'] = params[u'label'].encode("ascii")
+    newentity['id'] = 'e' + random_hex_generator(4)
+    newentity['agent_id'] = params['agent_id'] or random_hex_generator(12)
+    newentity['created_at'] = time.time()
+    newentity['updated_at'] = time.time()
```

Comment:
Similarly, any time you want two times to be the same, call `clock.seconds()` *once*; if you call it twice in rapid succession like this, 99.999% of the time the values will be the same, and then that one very very rare case where they're not exactly the same will cause confusion.